techtonica-projects/recipe-page/styles.css

```diff
@@ -129,6 +129,7 @@ h1:hover {
 @media (min-width: 1025px) {
   h1 {
       font-size: 2em;
+      /* test color: green; */
```

Comment:
I am assuming this commit is to convey you've attempted media queries testing.

Reply:
Yes because the font size change was so minimal I wanted to see the change more clearly.